openwisp_utils/admin_theme/views.py

```diff
@@ -21,7 +21,11 @@ def get(self, request, *args, **kwargs):
         context = self.get_context_data()
         # Add option for filtering objects with None field.
         results = []
-        if not self.term or self.term == '-':
+        if (
+            getattr(self.source_field, 'null', False)
```

Comment:
@nemesisdesign this condition check if the field is nullable

Reply:
great, thanks!